torch_geometric/typing.py

```diff
@@ -1,5 +1,7 @@
 from typing import Dict, List, Optional, Tuple, Union
 
+import numpy
```

Comment:
```suggestion
import numpy as np
```

Reply:
Why? We only use it once (`numpy.ndarray`)